docs/user-guide/commands/argocd_appset_generate.md

```diff
@@ -2,7 +2,7 @@
 
 ## argocd appset generate
 
-Generate apps of ApplicationSet rendered templates
+Generate apps from ApplicationSet rendered templates and display the output.
```

Comment:
This file is generated, so you'll have to find the old description in the CLI go code and update it there as well.

Reply:
Or simply run the `make codgen-local` command :)